cmd/cloud/cluster.go

```diff
@@ -28,11 +33,15 @@ var clusterCreateCmd = &cobra.Command{
 	Short: "Create a cluster.",
 	RunE: func(command *cobra.Command, args []string) error {
 		provider, _ := command.Flags().GetString("provider")
+		s3StateStore, _ := command.Flags().GetString("state-store")
 		size, _ := command.Flags().GetString("size")
+		zones, _ := command.Flags().GetString("zones")
+
+		splitZones := strings.Split(zones, ",")
```

Comment:
I like that this mimics the kops input-style of `--zones us-west-2a,us-west-2b,us-west-2c`. Might we also support `--zone us-west-2a --zone us-west-2b --zone us-west-2c`? If we keep `--zones` with its `comma-splitting` semantics, lets also document that in the corresponding flag help text.